module_11/index.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+include "data/users.php";
+include "data/passwords.php";
+include "data/authtorization.php";
+
+
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link href="styles.css" rel="stylesheet">
+    <title>Project - ведение списков</title>
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+</head>
+
+<body>
+
+<div class="header">
+    <div class="logo"><img src="i/logo.png" alt="Project"></div>
+    <div class="author">Автор: <span class="author__name">***</span></div>
+</div>
+
+<div class="clear">
+    <ul class="main-menu">
+        <li><a href="#">Главная</a></li>
+        <li><a href="#">О нас</a></li>
+        <li><a href="#">Контакты</a></li>
+        <li><a href="#">Новости</a></li>
+        <li><a href="#">Каталог</a></li>
+        <li><a href="/?login=yes"><?= $authorization ?></a></li>
```

Comment:
I would suggest you won't use additional variable for the message on the munu item. You can check state of authorisation here and display correct message. 
```
$authorised ? Выйти : Авторизация
```